cli/dcoscli/config/main.py

```diff
@@ -119,19 +188,35 @@ def _show(name):
     toml_config = config.get_config(True)
 
     if name is not None:
-        value = toml_config.get(name)
-        if value is None:
+        file_value = toml_config.get(name)
+        try:
+            # If the user presented a partial key name, eg 'core' when
+            # we have 'core.xyz'; we will get an exception here
+            effective_value, envvar_name = config.get_config_val_and_envvar(name)
```

Comment:
Yes. How about `get_config_val_info`? You can also use a continuation character ('\')